src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/resilience/RetriesSpec.groovy

```diff
@@ -592,7 +592,8 @@ and at least 1 path must remain safe"
         swDown = false
 
         then: "Flow goes UP"
-        wait(WAIT_OFFSET) { assert northboundV2.getFlowStatus(flow.flowId).status == FlowState.UP }
+        //sometimes flow is rerouted with ignore_bandwidth, then flow is DEGRADED
+        wait(WAIT_OFFSET) { assert northboundV2.getFlowStatus(flow.flowId).status in [FlowState.UP, FlowState.DEGRADED] }
```

Comment:
Add low_prio to this test please, since it tests single-multitable transition scenario, which we consider unimportant now

Reply:
added